galasa-parent/dev.galasa.framework.api.ras/openapi.yaml

```diff
@@ -310,3 +310,31 @@ components:
             type: string
           bundle:
             type: string
+    Run:
```

Comment:
Need the following fields,
bundle, shortTestClass, requestor, status, result, queued.